mocks/mockData.js

```diff
@@ -183,37 +184,53 @@ const people = [
   },
 ];
 
-const userId = "9e47cf29-1598-4269-aa31-db1d2e4e0207";
+const userId = '9e47cf29-1598-4269-aa31-db1d2e4e0207';
 
 const shiftTimeEntry = createTimeEntry({
   id: 'c0a80040-82cf-1986-8182-cfedbbd50003',
   timePeriodTypeId: '00000000-0000-0000-0000-000000000001',
-  actualStartTime: '2022-08-24T12:01:43.786+00:00',
-  actualEndTime: '2022-08-24T22:01:43.786+00:00',
+  actualStartTime: dayjs().hour(12).minute(0).format('YYYY-MM-DDTHH:mm:ssZ'),
+  actualEndTime: dayjs().hour(22).minute(0).format('YYYY-MM-DDTHH:mm:ssZ'),
```

Comment:
I’m interested in the dayjs usage here. How come you get away with not setting a day? Does it default to today’s date? I guess that’s pretty neat. Don’t want to have to be clicking back ages to see mock data. Cool 👍 (if my assumption is correct)

Reply:
Yeah `dayjs()` defaults to today. The mock data shows on every page no matter the date (which is also how it was working when it was set to 24th August), but before we made this change, we weren't getting any clashes because the dates were different